src/NumberField/NumberField.tsx

```diff
@@ -52,6 +119,58 @@ export default function NumberField(props: NumberFieldProps) {
             }
       }
       type="number"
+      value={currentValue}
     />
   );
 }
+
+/**
+ * Converts a string to a number.
+ * @param {*} value - The value to convert.
+ * @returns {number} - The converted value.
```

Comment:
Don't add types to jsdoc
```suggestion
 * @param value - The value to convert.
 * @returns - The converted value (either a number of null)
```

Reply:
Same point applies to all of these utility methods.